src/main/java/animals/model/Node.java

```diff
@@ -0,0 +1,52 @@
+package animals.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import static animals.service.ResourceBundleService.getLocalString;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Node {
+    private boolean isLeaf;
```

Comment:
Why do we need `isLeaf` as a field and not as a method as it was provided in the documentation?